src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function add(a, b) {
+    return a + b;
+  }
+
+  function subtract(a, b) {
+    return a - b;
+  }
+
+  function multiply(a, b) {
+    return a * b;
+  }
+
+  function divide(a, b) {
+    return a / b;
+  }
+
+  function operate(callback, num) {
+    result = callback(result, num);
+
+    return this;
+  }
+
+  function reset() {
+    result = 0;
+
+    return this;
+  }
+
+  return {
+    get result() {
+      return result;
+    },
+    add,
+    subtract,
+    multiply,
+    divide,
+    reset,
+    operate,
```

Comment:
The methods `add`, `subtract`, `multiply`, `divide`, `operate`, and `reset` should be defined using the method shorthand syntax instead of the function keyword.